### ISSUE TITLE
Can't install bbb-libreoffice-docker

### ISSUE THREAD

**Issue by qdzmc** [bigbluebutton/bigbluebutton#11214]
**Describe the bug**

**Installation type**
Did you install manually or did you use the bbb-install script?
use the bbb-install
**Firewall and IP address type**
Is your server behind a NAT or firewall? Does your BBB server have its own IPv4 addres?
 own IPv4 addres
**Console output**
Please include the full console output from the install.
wget -qO- https://ubuntu.bigbluebutton.org/bbb-install.sh | bash -s -- -v bionic-230-dev -s b.wue.cool
...
Get:194 http://deb.debian.org/debian buster/main amd64 libreoffice amd64 1:6.1.5-3+deb10u6 [78.1 kB]
E: Failed to fetch http://deb.debian.org/debian/pool/main/s/systemd/systemd_241-7~deb10u4_amd64.deb  404  Not Found [IP: 151.101.18.132 80]
E: Failed to fetch http://deb.debian.org/debian/pool/main/s/systemd/systemd-sysv_241-7~deb10u4_amd64.deb  404  Not Found [IP: 151.101.18.132 80]
E: Unable to fetch some archives, maybe run apt-get update or try with --fix-missing?
Fetched 241 MB in 50s (4807 kB/s)
The command '/bin/sh -c apt -y install  --no-install-recommends libreoffice fonts-crosextra-carlito fonts-crosextra-caladea fonts-noto fonts-noto-cjk' returned a non-zero code: 100
dpkg: error processing package bbb-libreoffice-docker (--configure):
 installed bbb-libreoffice-docker package post-installation script subprocess returned error exit status 100
Errors were encountered while processing:
 bbb-libreoffice-docker
E: Sub-process /usr/bin/dpkg returned an error code (1)
...
**Additional context**

bbb can't convert the .pptx file  but pdf is ok

**Comment by basisbit** [comment]
For support requests, please use the [forum / mailing list](https://groups.google.com/forum/#!forum/bigbluebutton-setup). This issue tracker here is only for BBB development related issues. Sorry.

TLDR: this is not a problem in BigBlueButton, but instead probably is a bug in your network setup or you forgot to run `apt update` and `apt upgrade` on your system.
